common/txmgr/types/tx_store.go

```diff
@@ -35,8 +35,10 @@ type TxStore[
 	TxHistoryReaper[CHAIN_ID]
 	TransactionStore[ADDR, CHAIN_ID, TX_HASH, BLOCK_HASH, SEQ, FEE]
 
-	// methods for saving & retreiving receipts
-	FindReceiptsPendingConfirmation(ctx context.Context, blockNum int64, chainID CHAIN_ID) (receiptsPlus []ReceiptPlus[R], err error)
+	// Find confirmed txes requiring callback but have not yet been signaled
```

Comment:
nit: mention that these are confirrmed, beyond the minConfirmations param. Indicates that these are kind of finalized.